src/WebUI/src/services/characters-service.spec.ts

```diff
@@ -146,7 +150,7 @@ it('updateCharacterCharacteristics', async () => {
 it.each([
   [0, 0],
   [1, 0],
-  [2, 3538],
+  [2, 388],
```

Comment:
changed after an [edit](https://github.com/verdie-g/crpg/commit/61aa77967ff57ede4d6c09597d65ec0e8efd271d)

btw, we can store shared test-cases for these things in json(json5) format